src/command.c

```diff
@@ -10776,8 +10776,7 @@ int GetUserHelp (char *helpTkn)
         MrBayesPrint ("                    both rates. (The first number is off->on and the second      \n");
         MrBayesPrint ("                    is on->off).                                                 \n");
         MrBayesPrint ("   Symdirihyperpr - This option sets the prior for the stationary frequencies    \n");
-        MrBayesPrint ("                    of the states for morphological (standard) data. There can   \n");
-        MrBayesPrint ("                    be as many as 10 states for standard data. However, the      \n");
+        MrBayesPrint ("                    of the states for morphological (standard) data. The         \n");
```

Comment:
I think it would be good to state here the maximum number of states that can be used, since this is hardset.

Reply:
I think there is a message elsewhere indicating the max number of states. it is not needed here